lib/matplotlib/__init__.py

```diff
@@ -1570,7 +1570,15 @@ def _init_tests():
         warnings.warn(
             "matplotlib is not built with the correct FreeType version to run "
             "tests.  Set local_freetype=True in setup.cfg and rebuild. "
-            "Expect many image comparison failures below.")
+            "Expect many image comparison failures below. "
+            "Expected freetype version {0}. "
+            "Found freetype version {1}."
```

Comment:
What was not the case in the original? The original had spaces between sentences. This one is missing a space between two sentences.